packages/near-membrane-base/src/membrane.ts

```diff
@@ -1644,4 +1644,15 @@ export function createMembraneMarshall() {
     };
 }
 
-export const marshallSourceTextInStrictMode = `(function(){'use strict';return (${createMembraneMarshall.toString()})})()`;
+const TypeErrorCtor = TypeError;
```

Comment:
If you need this, this must be cached at the top as part of the outer layer (the marshall)

Reply:
It's only used in blue realm execution